### DISCUSSION TITLE
The type placeholder `_` is not allowed within types on item signatures [E0121]

### DISCUSSION THREAD

**Discussion by averichev** [rwf2/Rocket#2035] [Questions]
Hello one!

My Rider inspection show problem in `fn rocket()`

**_The type placeholder `_` is not allowed within types on item signatures [E0121]_**

How I can fix it?

![image](https://user-images.githubusercontent.com/5820708/147328271-c746acfb-f550-47dc-9abd-a8f4996d9de2.png)

**Comment by nathanielford** [comment]
This is a bug in `intellij-rust` and has an [open issue](https://github.com/intellij-rust/intellij-rust/issues/5975).
